src/tools/wasm-ctor-eval.cpp

```diff
@@ -369,7 +369,12 @@ int main(int argc, const char* argv[]) {
       .add("--ctors", "-c", "Comma-separated list of global constructor functions to evaluate",
            Options::Arguments::One,
            [&](Options* o, const std::string& argument) {
-             ctorsString = argument;
+             if (!argument.empty() && argument[0] == '@'){
```

Comment:
I meant to put this logic into the utility function `read_response_file`. That way it's easy for other options to support response files too.